package.json

```diff
@@ -177,6 +192,16 @@
                     "command": "deploymentSlot.SwapSlots",
                     "when": "view == azureAppService && viewItem == deploymentSlot",
                     "group": "2_deploymentSlotGeneralCommands@2"
+                },
+                {
+                    "command": "webJobs.OpenInPortal",
```

Comment:
Can we combine the OpenInPortal commands into one? If you open up the command pallete, you see it 4 times :/
![screen shot 2017-09-14 at 3 38 20 pm](https://user-images.githubusercontent.com/11282622/30458977-d6cbf5be-9962-11e7-95b3-9122d4322f63.png)

Reply:
Just a few ideas: You could create an interface or something for all Azure nodes that can be opened in the portal. Or you could add an optional function 'OpenInPortal' to NodeBase.